File: SCG Addon Taller/DataContract/Vehiculos/AccesoriosXVehiculo.vb
Public Class AccesoriosxVehiculo


    Private _LogInst As Int32
    Public Property LogInst() As Int32
        Get
            Return _LogInst
        End Get
        Set(value As Int32)
            _LogInst = value
        End Set
    End Property


    Private _LineId As Int32
    Public Property LineId() As Int32
        Get
            Return _LineId
        End Get
        Set(value As Int32)
            _LineId = value
        End Set
    End Property


    Private _Code As String
    Public Property Code() As String
        Get
            Return _Code
        End Get
        Set(value As String)
            _Code = value
        End Set
    End Property


    Private _U_Acc As String
    Public Property U_Acc() As String
        Get
            Return _U_Acc
        End Get
        Set(value As String)
            _U_Acc = value
        End Set
    End Property


    Private _U_N_Acc As String
    Public Property U_N_Acc() As String
        Get
            Return _U_N_Acc
        End Get
        Set(value As String)
            _U_N_Acc = value
        End Set
    End Property


    Private _U_Tipo As String
    Public Property U_Tipo() As String
        Get
            Return _U_Tipo
        End Get
        Set(value As String)
            _U_Tipo = value
        End Set
    End Property

End Class

File: DMS_Connector/Business Logic/DataContract/Vehiculos/AccesoriosXVehiculo.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DMS_Connector.Business_Logic.DataContract.Vehiculos
{
    public class AccesoriosXVehiculo
    {
        public String Code { get; set; }
        public Int32 LineId { get; set; }
        public Int32? LogInst { get; set; }
        public String U_Acc { get; set; }
        public String U_N_Acc { get; set; }
        public String U_Tipo { get; set; }
    }
}
